Dockerfile

```diff
@@ -0,0 +1,31 @@
+#base images
+FROM ubuntu:18.04
+FROM python:3.8
+
+#copy env file
+WORKDIR /src
+
+#install dependencies    
+RUN pip install aif360 && \
```

Comment:
you are install aif360 multiple times in this line.

Reply:
Thanks I have remove the duplicate install command from dockerfile